src/styles.css

```diff
@@ -322,23 +326,27 @@ select {
 
 .footerReg {
   background-color: rgb(100, 100, 100);
-  padding-top: 40px;
+  display: flex;
+  flex-flow: column;
 }
 
 .footerContent {
   color: rgb(255, 255, 255);
   text-align: center;
-  font-size: 14px;
-  max-width: 760px;
+  font-size: 1.4rem;
+  max-width: 76rem;
+  align-self: center;
+  margin: 0 0 3rem;
 }
 
 .smallHeading {
-  font-size: 16px;
+  font-size: 1.6rem;
+  padding: 2.5rem 0;
 }
 
 .paymentCards {
-  max-height: 40px;
-  margin-top: 20px;
+  max-height: 4rem;

```

Comment:
when you use min/max that means you want that to be fixed, with rem it's relative, so you shouldn't use rem there